pages/api/profiles/[username]/index.js

```diff
@@ -114,6 +114,9 @@ export async function getUserApi(req, res, username, options = {}) {
       ),
     socials: getProfile.links
       .filter((link) => link.isPinned)
+      .sort(
+        (a, b) => (a.order ?? Number.MAX_VALUE) - (b.order ?? Number.MAX_VALUE),
```

Comment:
Is this ```?? Number.MAX_VALUE``` really required can you please explain what edge case does it handle.

Reply:
`Number.MAX_VALUE` is used in case a.order or b.order values are null or undefined. You can see the same method used for the enabled links part right above this code.

Reply:
Great collaboration 👍